tests/unit/backend/test_common.py

```diff
@@ -59,14 +59,22 @@ def test_string_include(self):
         PyRDF.include_headers("tests/unit/backend/test_headers/header1.hxx")
 
         required_header = ["tests/unit/backend/test_headers/header1.hxx"]
-        self.assertListEqual(PyRDF.includes_headers, required_header)
+        # Feature detection: first try Python 3 function, then Python 2
```

Comment:
For tests we can just convert `PyRDF.includes_headers` to a list since it makes the code way simpler:

```
 self.assertListEqual(list(PyRDF.includes_headers), required_header)
```

Same for the rest.

Reply:
this would not guarantee the same order of the elements